content/en/news/releases/1.9.x/announcing-1.9.4/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Announcing Istio 1.9.4
+linktitle: 1.9.4
+subtitle: Patch Release
+description: Istio 1.9.4 patch release.
+publishdate: 2020-04-27
+release: 1.9.4
+aliases:
+    - /news/announcing-1.9.4
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.9.3 and Istio 1.9.4
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue where the Istio operator prunes the resource that doesn't belong to specific Istio operator CR. ([Issue #30833](https://github.com/istio/istio/issues/30833))
```

Comment:
```suggestion
- **Fixed** an issue where the Istio operator prunes any resources that don't belong to a specific Istio operator custom resource. ([Issue #30833](https://github.com/istio/istio/issues/30833))
```

I'm still not sure if this expresses the same behavior described here though: https://github.com/istio/istio/issues/30833#issuecomment-823728149

Reply:
I think what I'm understanding from that comment in the PR is: 

```
- **Fixed** an issue where the Istio operator prunes all resources created by the operator, including itself. Now the operator will only remove resources belonging to the custom resource.
```

Reply:
@morvencao can you please take a look?

Reply:
Correct, the description looks good to me, with the fix, istio operator will only prune the resources for enabled components in the CR.

Reply:
Thanks